src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -3,17 +3,33 @@
     $connected_entity = new $field['model'];
     $connected_entity_key_name = $connected_entity->getKeyName();
     $old_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+
+    $allows_null = $crud->model::isColumnNullable($field['name']) ?
+        ((isset($field['allows_null']) && $field['allows_null'] != false) || !isset($field['allows_null']) ? true : false) :
+        ((isset($field['allows_null']) && $field['allows_null'] != true) || !isset($field['allows_null']) ? false : true);
+
```

Comment:
I feel like the above can be simplified by using:

```php
$field['allows_null'] = isset($field['allows_null']) ? (bool)$field['allows_null'] : $crud->model::isColumnNullable($field['name']);
$allows_null = $field['allows_null'];
```

I think it would achieve the same thing, right? Return ```isColumnNullable()```, unless the developer has overwritten ```allows_null```, in which case use whatever the developer wanted (but cast it to boolean so that ```"false"``` gets treated as ```false``` and stuff like that).